### ISSUE TITLE
Cannot execution evaluate

### ISSUE THREAD

**Issue by rickysuperpower** [facebookresearch/demucs#578]
## ❓ Questions

I try to train Demucs and finish training 20 epochs. But when it starts evaluate models , my gpu and cpu stop with no any error messages. Could someone help me figure out the issue? Any advice is appreciated.

**Comment by CarlGao4** [comment]
Are you training the original four-stem model?

**Comment by rickysuperpower** [comment]
Yes, I used MUSDBHQ to train demucs. I don't think the training is difficult, but my computer cannot move.

**Comment by CarlGao4** [comment]
Are you sure it isn't killed?

**Comment by rickysuperpower** [comment]
Yes. My GPUs still have more capacity but they didn't move any more. And there were no error messages. 
![スクリーンショット 2023-12-22 112927](https://github.com/facebookresearch/demucs/assets/131159826/74c117cc-f2c8-4745-b73c-c08cdedabed7)

**Comment by QKTheoNguyen** [comment]
Hello, I have the same problem, did you overcome it yet ?
